_docs/_latest/setup/install.md

```diff
@@ -98,8 +98,8 @@ Create a file named `main.tf` in an empty directory and copy the contents below
 ```hcl
 module "forseti" {
   source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.1"
-
+  version = "~> 5.2"
```

Comment:
The version for the Forseti module should be 5.1.

Reply:
Ok I just realised that latest here is for 2.25 - are we supposed to build 2.25 out if we have not released it yet?

Reply:
Website updates and release 2.25 will go out at the same time (most likely next week).  I updated to 2.25 to avoid coming back to this in a week.